plugins/woocommerce/i18n/states.php

```diff
@@ -321,6 +321,41 @@
 		'CN31' => __( 'Tibet / 西藏', 'woocommerce' ),
 		'CN32' => __( 'Xinjiang / 新疆', 'woocommerce' ),
 	),
+	'CO' => array( // Colombia States.
+		'CO-DC' => __( 'Distrito Capital de Bogotá', 'woocommerce' ),
```

Comment:
This should be [Capital District](https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L894) according to the CLDR.